typings/definitions.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference path='tsd.d.ts' />
```

Comment:
Similar reasoning to my comment above - if `lib/` had more hierarchy, each directory would have a `typings` directory with `definitions.d.ts` that would contain at least:
```ts
/// <reference path='../../typings/defintions.d.ts' />
```

To refer to the parent directory's typings.